docs/ja/editorialguideline.md

```diff
@@ -1,155 +1,173 @@
+# ガイドラインの適用範囲
```

Comment:
The first line (h1) should be the title of the page, i.e. the equivalent of "Editorial Guidelines".

All the other title should move down one level (h1 -> h2, h2 -> h3).